### ISSUE TITLE
Gulp task doesn't copy angular.min.js and angular-animate.min.js

### ISSUE THREAD

**Issue by memee** [nikhilmodak/gulp-ngdocs#90]
Default gulp task example doesn't copy angular files into docs/js.
Only `angular-bootstrap[-prettify].js` are copied. Tested with `gulp-ngdocs@0.2.13`

**Comment by rdkleine** [comment]
Same here.

**Comment by cpasslack** [comment]
I have the same problem

**Comment by RobinWest** [comment]
Yeah got me too. This issue has been fixed but if you're using npm you might be having some problems still. Please see #80.

**Comment by huerlisi** [comment]
A new release has been made a couple days ago with support for npm 3. Please test:-)

**Comment by memee** [comment]
+1
